scripts/cnode-helper-scripts/env

```diff
@@ -106,19 +107,26 @@ telegramSend() {
   fi
 }
 
+[[ -z "${CNODEBIN}" ]] && CNODEBIN=$(command -v cardano-node)
+if [[ -z "${CNODEBIN}" ]]; then
+  if [[ -f "${HOME}/.cabal/bin/cardano-node" ]]; then
+    export PATH="${HOME}/.cabal/bin":$PATH
+    CNODEBIN=$(command -v cardano-node)
+  else
+    echo "You do not have a cardano-node binary available in \$PATH."
+    return 1
+  fi
+fi
+
 [[ -z "${CCLI}" ]] && CCLI=$(command -v cardano-cli)
 if [[ -z "${CCLI}" ]]; then
   if [[ -f "${HOME}/.cabal/bin/cardano-cli" ]]; then
-    # Assumption being made that cardano-cli and cardano-node are both present, if not - prereqs and build instructions were not followed
     export PATH="${HOME}/.cabal/bin":$PATH
     CCLI=$(command -v cardano-cli)
   else
     echo "You do not have a cardano-cli binary available in \$PATH."
     return 1
   fi
-else
```

Comment:
We check `command -v` execution above, if not empty - the CCLI_PARENT is already present in ${PATH} . Hence, removed the else condition